src/features/documents/DocumentList.tsx

```diff
@@ -1,23 +1,22 @@
-import React, { useCallback, useEffect } from 'react';
+import React, { useState, useCallback, useEffect } from 'react';
 import { useParams, useNavigate, useLocation } from 'react-router-dom';
 import * as moment from 'moment';
 
 import { useAppDispatch, useAppSelector } from 'app/hooks';
-import { selectDocumentList, listDocumentsAsync } from './documentsSlice';
+import { selectDocumentList, listDocumentsAsync, searchDocumentsAsync } from './documentsSlice';
 
 export function DocumentList() {
-  const { documents, hasPrevious, hasNext, status } = useAppSelector(selectDocumentList);
-  const { projectName } = useParams();
-  const location = useLocation();
-  const navigate = useNavigate();
   const dispatch = useAppDispatch();
+  const documentKey = useLocation().pathname.replace(/^\/projects\/(.*)\/documents\//, '');
```

Comment:
Can we use `useParams` here?
```suggestion
  const { projectName, documentKey } = useParams();
```

https://github.com/yorkie-team/yorkie-house/blob/main/src/features/documents/DocumentDetail.tsx#L2

Reply:
The `useParams` hook returns an object of key/value pairs of the dynamic params 
from the current URL that was matched by the `<Route path>`.

So I can use `const documentKey = useParams()['*']` in DocumentList component.
`<Route path="/projects/:projectName/documents/*" element={<Documents />} />`